src/test/regress/specs/isolation_distributed_transaction_id.spec

```diff
@@ -57,6 +62,11 @@ step "s2-assign-transaction-id"
     SELECT assign_distributed_transaction_id(2, 2, '2015-01-02 00:00:00+0');
 }
 
+step "s2-vacuum"
+{
+	VACUUM FULL pg_dist_partition;
```

Comment:
Verified that this is failing on master.

What other commands can lead to this situation? I'd like to check that the issue we're seeing in the travis outputs will be solved by this. I only see create/drop schema-view-type-tables in the concurrent tests, which doesn't seem to lead the issue?

Reply:
It seems it's usually triggered by autovacuum.